README.md

```diff
@@ -38,6 +44,22 @@ To provide additional information about the accuracy of the result in addition t
 <img alt="A line graph showing a cross-correlation curve with a sharp prominent peak emerging from low-level noise.  A dotted vertical line is overlaid at the position of the peak, indicating the position of the calculated offset." src="example_plot.png" title="Example correlation plot" />
 </div>
 
+Library Usage
+-------------
+
+To use the Python library:
+
+```python
+from audio_offset_finder.audio_offset_finder import find_offset_between_files
```

Comment:
DId you consider having just one `find_offset` function but make it determine whether the input is a filename or a buffer dynamically?

Reply:
This all sounds good to me!